nvpair/src/ser.rs

```diff
@@ -7,11 +7,11 @@ struct Serializer {}
 
 #[derive(Debug)]
 enum NvDataOwned {
-    Unknown,
+    //Unknown,
     None,
     Bool,
     BoolV(bool),
-    Byte(u8),
+    //Byte(u8),
```

Comment:
Why are we removing Byte and Unknown here? Does their implementation break something?

Reply:
They were unused, which Clippy complained about.

Reply:
Maybe I should have removed them entirely instead of just commenting out?

Reply:
I think it's fine as is; we may want them later, so leaving the code around feels acceptable to me.